backend/lru-cache.js

```diff
@@ -0,0 +1,24 @@
+// How many entries to keep at most.
+const MAX_CACHE_ENTRIES = 1000
```

Comment:
Note: if we end up going with this (I need to run some benchmarks to figure this out), we'll want to make this a customizable option that you can pass in to `createCache`.

Also this file (like all js files) should have `'use strict'` at the top of the file.